submissions/Shizopo/html-css-popup/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="./styles/index.css">
+    <title>Wooble</title>
+</head>
+<body>
+    <div class="header">
+        <a href="#" class="header-link" title="Your light and secure mail">Wmail</a>
+        <a href="#" class="header-link" title="Your photo gallery">Photos</a>
+        <input type="checkbox" id="apps-checkbox" class="apps-menu-checkbox visually-hidden">
+        <label for="apps-checkbox" class="apps-menu-button" title="Your Wooble apps">
+            <img src="./img/menu-icon.svg" class="apps-menu-icon">
+        </label>
+        <div class="apps">
+            <ul class="apps-list">
+                <li>
+                    <a href="#" class="app-link" title="Mail">
+                        <div class="app-link-logo">M</div>
```

Comment:
Why you use `div` in this place and into other similar places you use `span`?

Reply:
My bad! Unfortunately missed it. Now it's fixed, see commits below.